chevah/keycert/ssh.py

```diff
@@ -690,6 +697,8 @@ def _fromString_PUBLIC_OPENSSH(cls, data):
         @raises BadKeyError: if the blob type is unknown.
         """
         blob = base64.decodestring(data.split()[1])
+        if cls._guessStringType(blob) != 'blob':
+            raise BadKeyError("Fail to parse OpenSSH key content.")
```

Comment:
There were a couple of good blog post on the internet describing why to many if statements are bad practice, but I can not find them right now.... but basically, less is more... and if you can get the same/similar with less if statements, that solution is better :)